.travis.yml

```diff
@@ -5,7 +5,14 @@ python:
   - 3.4
   - 3.5
   - 3.6
-  - pypy-5.4.1
+  - pypy
+  - pypy3
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
```

Comment:
Random observation about something I learned recently: you need `dist: xenial` because OpenSSL in trusty is too old (Python 3.7 requires OpenSSL 1.1), and you need `sudo: true` because the container-based Travis builds currently ignore `dist:` entirely and always runs trusty.  Travis CI plans to get rid of the entire container-based infrastructure in the nearish future (~ several months AFAIU), because they find it a pain to manage two parallel infrastructures, and the container-based one causes problems for people who want to use Docker in their builds.

I believe today `sudo: true` is default unless you explicitly specify `sudo: false` (like we do, since Travis recommended that at one point saying you'd get faster builds that way!), and at some point in the future `sudo: false` is going to be ignored.

At some point in the future (July 2019) `trusty` is going to be EOLed and I think/hope Travis will switch to dist: xenial by default.  And then finally we'll be able to ask for Python 3.7 without any special hacks!